### PR TITLE
Convert `for...in` to `each` in views

### PR DESCRIPTION
**Pull request by nimmolo**
One good thing came out of running Brakeman locally - it refused to parse the `for` loops in the views. So here you go... `for` loops are converted to `each` loops.

### CODE REVIEW

File: app/views/species_list/list_species_lists.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <%= paginate_block(@pages) do %>
   <% if @objects.any? %>
     <div class="list-group">
-      <% for species_list in @objects.each %>
```
Comment (nimmolo) [inline app/views/species_list/list_species_lists.html.erb:13]: Not sure I did this one right. Are these equivalent?

Comment (nimmolo) [reply app/views/species_list/list_species_lists.html.erb:13]: After researching a bit, i'm pretty sure these are equivalent.
`for species_list in @objects.each`
`@objects.each do |species_list|`

**Comment by pellaea** [comment]
The only difference, as Joe explained it a short while ago, is subtle: in
the case of the for loop, the accumulator (species_list in your example)
and any local variables used in the body of the for loop are visible
outside the loop.  In the case of the each construct, they are not visible
outside the each block.

But as I was blissfully unaware of this dubious "feature" of for loops, I
never relied on it in my code, so it should not be an issue.

File: app/views/species_list/list_species_lists.html.erb (hunk 1 above)
Comment (JoeCohen) [reply app/views/species_list/list_species_lists.html.erb:13]: Looks right.

**Comment by nimmolo** [comment]
Thanks for the reviews. Will merge now.

**Review by JoeCohen — APPROVED** [review]
Looks good to me. That's a lot of work, but might as well get it right and be consistent with `app`.
NOTE: As a matter of principle, Rubocop refuses to look at `.erb` files. https://github.com/rubocop/rubocop/issues/1113
I wonder if there are other potential problems lurking there?
Maybe down the line we should add a `.erb` linter?

**Pull request merged**